hbw/app/javascript/packs/hbw/components/form/string.js.jsx

```diff
@@ -270,6 +270,8 @@ modulejs.define('HBWFormString', ['React'], (React) => {
           visualValue: $el.value
         });
       }
+
+      this.props.trigger('hbw:update-value', { name: this.props.name, value: this.state.value });
```

Comment:
Let's make the function in the `WithCondition` HOC, something like `fireFieldValueUpdate(name, value)`.